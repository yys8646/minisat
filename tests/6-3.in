6
11....
......
.....1
.0.0..
.....1
.11...
