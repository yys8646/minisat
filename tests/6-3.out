6
110100
001101
010011
101010
100101
011010
